src/styles/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: auto auto 40px;
+  }
+
+  &__title {
+    color: $color-main;
```

Comment:
The variable `$color-main` is used here but not defined in this file. Make sure it is defined in a variables file or elsewhere in your SCSS setup.